modules/stac-server/opensearch_serverless.tf

```diff
@@ -98,6 +98,51 @@ resource "aws_opensearchserverless_collection" "stac_server_opensearch_serverles
   ]
 }
 
+resource "aws_lambda_function" "stac_server_waiting_for_opensearch_serverless_active_collections" {
+  count            = var.deploy_stac_server_opensearch_serverless ? 1 : 0
+  filename         = data.archive_file.waiting_for_opensearch_lambda_zip.output_path
+  source_code_hash = data.archive_file.waiting_for_opensearch_lambda_zip.output_base64sha256
+  function_name    = "${local.name_prefix}-stac-server-oss-wait-collections"
+  role             = aws_iam_role.stac_api_lambda_role.arn
+  description      = "Lambda function to wait for Opensearch Serverless Collection to be active."
```

Comment:
Maybe "Polls an opensearch serverless collection to ensure it is active prior to attempting to ingest."

Reply:
Changed description